packages/apps/workshop/stories/design-system/components/message-webcomponent.stories.js

```diff
@@ -21,49 +21,68 @@ export default {
   },
 };
 
-export const Normal = forModule(moduleName).createElement(
+export const Default = forModule(moduleName).createElement(
   () => compileTemplate(
     `
-    <oui-message type="info">Message</oui-message>
-    <oui-message type="success">Message</oui-message>
-    <oui-message type="warning">Message</oui-message>
-    <oui-message type="error">Message</oui-message>`,
+    <!-- Info -->
+    <oui-message type="info">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
Nice to have a longer and therefore more realistic message

Reply:
We try to reflect the examples from the UI specs. But if our Storybook examples aren't practical, we can still improve them :)
For example, having a knob to add the possibility to edit directly the message content.